src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -311,6 +311,142 @@ def k8s_install_cli(client_version='latest', install_location=None):
         raise CLIError('Connection error while attempting to download client ({})'.format(ex))
 
 
+def k8s_install_connector(client, name, resource_group, connector_name,
+                          location=None, service_principal=None, client_secret=None,
+                          chart_url=None, os_type='Linux'):
+    """Deploy the ACI-Connector to a AKS cluster
+    :param name: The name of the AKS cluster. The name is case insensitive.
+    :type name: str
+    :param connector_name: The name for the ACI Connector
+    :type connector_name: str
+    :param resource_group: Name of the resource group where the AKS is located and where
+    the ACI connector will be mapped.
+    :type resource_group: str
+    :param location:
+    :type location: str
+    :param service_principal: The service principal used for ACI authentication
+    to Azure APIs. If not specified, it is created for you and stored in the
+    ${HOME}/.azure directory.
+    :type service_principal: str
+    :param client_secret: The secret associated with the service principal. If
+    --service-principal is specified, then secret should also be specified. If
+    --service-principal is not specified, the secret is auto-generated for you
+    and stored in ${HOME}/.azure/ directory.
+    :type client_secret: str
+    :param chart_url: URL to the chart,
+    Default: https://github.com/Azure/aci-connector-k8s/raw/master/charts/aci-connector.tgz
+    :type chart_url: str
+    :param os_type: Os type target, could be Windows, Linux or Both.
+    :type os_type: str
+    """
```

Comment:
All help should be defined in params.py and _help.py not in the docstring of the commands.

`--os-type`: In params.py, define a choice_list of this param with possible values of ['Windows', 'Linux', 'Both']
`--chart-url`: In params.py, you can specify the default value as a string `default='...'`